<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="FileButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;browser file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>接收</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>141</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>24</y>
      <width>110</width>
      <height>145</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_4"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_5">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Old</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Even</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mark</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Space</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>检验位</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="currentText">
        <string>9600</string>
       </property>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>56000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128000</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>FileButton</zorder>
   <zorder>textBrowser</zorder>
   <zorder>textEdit</zorder>
   <zorder>pushButton</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>comboBox_3</tabstop>
  <tabstop>comboBox_4</tabstop>
  <tabstop>FileButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_2</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>textBrowser</receiver>
   <slot>append(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>textBrowser</receiver>
   <slot>append(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_3</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>textBrowser</receiver>
   <slot>append(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_5</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>textBrowser</receiver>
   <slot>append(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_4</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>textBrowser</receiver>
   <slot>append(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
